api/envoy/config/trace/v3/opentelemetry.proto

```diff
@@ -17,11 +21,41 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // Configuration for the OpenTelemetry tracer.
 //  [#extension: envoy.tracers.opentelemetry]
 message OpenTelemetryConfig {
+  // Configuration for sending traces to an upstream cluster over HTTP. Note that
+  // this will only be used if the grpc_service is not set.
+  // [#next-free-field: 6]
+  message HttpConfig {
+    // The upstream cluster that will receive OTLP traces over HTTP.
+    string cluster_name = 1;
+
+    // The path to the trace ingest endpoint. The path is appended with the host name configured in the cluster.
+    // Optional: If omitted, "/v1/traces" will be used as default.
+    string traces_path = 2;
+
+    // Sets the maximum duration in milliseconds that a response can take to arrive upon request.
+    google.protobuf.Duration timeout = 3 [(validate.rules).duration = {
+      required: true
+      gte {}
+    }];
+
+    // Custom headers (key-value) to be added to the OTLP HTTP request.
+    repeated core.v3.HeaderValue headers = 4;
+
+    // The hostname to include in the Host header of the OTLP HTTP request.
+    string hostname = 5;
+  }
+
   // The upstream gRPC cluster that will receive OTLP traces.
   // Note that the tracer drops traces if the server does not read data fast enough.
-  // This field can be left empty to disable reporting traces to the collector.
+  // This field can be left empty to disable reporting traces to the upstream cluster.
   core.v3.GrpcService grpc_service = 1;
```

Comment:
please add a comment here saying this is mutual exclusive with `http_service`. See example [here](https://github.com/envoyproxy/envoy/blob/403d1db30f55721eb98ee655e61b5e4c73da6645/api/envoy/extensions/transport_sockets/tls/v3/tls.proto#L247)

Reply:
Done! [f507a0f](https://github.com/envoyproxy/envoy/pull/29207/commits/f507a0f6b7f5e2f237d27572efa504b918f7f15f)